project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.9.7
+sbt.version=1.9.9
```

Comment:
I'm just curious is it right to use version 1.9.9 of sbt. You are probably right but I'm commenting that just to double-check

![image](https://github.com/hmrc/help-to-save/assets/112881943/b015014a-7ab1-46a1-8fa9-75e5f16ec66a)